### PR TITLE
[FIX] use "cls" instead of "self" for @classmethod

### PR DESCRIPTION
**Pull request by DimitriPapadopoulos**
This is a convention, not a requirement, but still...

### CODE REVIEW

File: nilearn/plotting/displays.py [hunk 1]
```diff
@@ -1897,17 +1897,17 @@ def find_cut_coords(self, img=None, threshold=None, cut_coords=None):
         if (not isinstance(cut_coords, collections.abc.Sequence) and
                 isinstance(cut_coords, numbers.Number)):
             cut_coords = [cut_coords] * 3
-            cut_coords = self._find_cut_coords(img, cut_coords,
-                                               self._cut_displayed)
+            cut_coords = cls._find_cut_coords(img, cut_coords,
```
Comment (jeromedockes) [inline nilearn/plotting/displays.py:1900]: thanks! for this to work we still need to make `_find_cut_coords` a static method

Comment (DimitriPapadopoulos) [reply nilearn/plotting/displays.py:1900]: Yes, see previous PR #2921.

Comment (DimitriPapadopoulos) [reply nilearn/plotting/displays.py:1900]: We do need to merge #2921 first.

Comment (DimitriPapadopoulos) [reply nilearn/plotting/displays.py:1900]: Actually, this works (not) the same way before/after the PR. The real fix is in #2921, this here is just a variable name change.

**Review by agramfort — APPROVED** [review]

**Comment by agramfort** [comment]
you have CI failures @DimitriPapadopoulos

**Comment by DimitriPapadopoulos** [comment]
Fixed now.

**Comment by agramfort** [comment]
@jeromedockes feel free to MRG

**Comment by jeromedockes** [comment]
great, thanks!

**Pull request merged**